transaction-service/src/main/java/pl/adamboguszewski/transaction/service/infrastructure/controller/TransactionController.java

```diff
@@ -59,8 +59,8 @@ public CreateTransactionResponse createTransaction(@Valid @RequestBody CreateTra
     }
 
     @DeleteMapping("/{id}")
-    public ResponseEntity deleteTransaction(@PathVariable Long id) {
+    public ResponseEntity<Optional<Transaction>> deleteTransaction(@PathVariable Long id) {
         transactionService.deleteTransaction(id);
-        return new ResponseEntity(HttpStatus.OK);
+        return new ResponseEntity<>(Optional.empty(), HttpStatus.OK);
```

Comment:
define Optional\<Transaction> just to return Optional.empty() 🤔 just return new ResponseEntity(HttpStatus.OK);